syz-verifier/verifier.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/google/syzkaller/pkg/mgrconfig"
+	"github.com/google/syzkaller/pkg/osutil"
+	"github.com/google/syzkaller/prog"
+	"github.com/google/syzkaller/vm"
+
+	"github.com/google/syzkaller/sys/targets"
+)
+
+type cfgFlagVals []string
+
+func (cfvs *cfgFlagVals) String() string {
+	return fmt.Sprint(*cfvs)
+}
+
+func (cfvs *cfgFlagVals) Set(value string) error {
```

Comment:
It's usually better to place more important things at the top of the file and less important/utility things (like cfgFlagVals) towards the bottom of the file. It allows for better readability, readers can read top-down and scroll naturally. C/C++ force the opposite order, but it's really just an implication how the compiler was structured back in 60s to save on 1 compiler pass. Overall if A uses B, then A should be defines before B.
Please move cfgFlagVals to the bottom of the file.